Sources/Testing/Events/Clock.swift

```diff
@@ -8,7 +8,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-@_implementationOnly import TestingInternals
+private import TestingInternals
```

Comment:
It's probably fine to always `internal` import TestingInternals.

Reply:
I like having `private` on those which are eligible, as a statement of intent, mainly. I feel like clarifying the role that the `TestingInternals` (or other modules) play can be useful to understand the effect of an `import` within a particular file